libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.BeginDialog.schema

```diff
@@ -12,6 +12,7 @@
         },
         "disabled": {
             "$role": "expression",
+            "type": [ "boolean", "string"],
```

Comment:
>"type": [ "boolean", "string"] [](start = 12, length = 30)

I suspect verify is going to barf on this.  I'm planning to look at updating it.